plugin.yaml

```diff
@@ -57,15 +48,48 @@ node_types:
           in default zone.
         type: string
         default: ''
+      tags:
+        description: >
+          Optional tags. If not given, this instance will have a tag only with
+          its name.
+        type: string
+        default: ''
+      block_project_ssh_keys:
+        description: >
+          Disable project-wide ssh keys for this Instance
+        type: boolean
+        default: false
+      can_ip_forward:
+        description: >
+          Is the VM allowed to send packets with source address different to its own?
+        type: boolean
+        default: false
+      scopes:
+        description: >
+          Optional scopes. If not will set by default: 
+          'https://www.googleapis.com/auth/devstorage.read_write',
+          'https://www.googleapis.com/auth/logging.write'
+        default: []
+      startup_script:
+        description: >
+          A script which will be run when the Instance is first started
+          example:
+            type: string
+            script: |
+              yum install some stuff
+              systemctl start it
+          or:
+            type: file
+            script: <path to script file>
+        default:
+          ''
     interfaces:
       cloudify.interfaces.lifecycle:
-        start:
-          implementation: gcp_plugin.gcp.compute.instance.create
+        configure:
```

Comment:
This is confusing. Surely create should be create (not sure why it's `configure` here?)?